example/vectorAdd/src/vectorAdd.cpp

```diff
@@ -182,13 +183,23 @@ auto main()
         numElements));
 
     // Enqueue the kernel execution task
-    alpaka::queue::enqueue(queue, taskKernel);
+    {
+        const auto beginT = std::chrono::high_resolution_clock::now();
+        alpaka::queue::enqueue(queue, taskKernel);
+        const auto endT = std::chrono::high_resolution_clock::now();
+        std::cout << "Time for kernel execution: " << std::chrono::duration<double>(endT-beginT).count() << 's' << std::endl;
+    }
 
     // Copy back the result
-    alpaka::mem::view::copy(queue, bufHostC, bufAccC, extent);
-    alpaka::wait::wait(queue);
+    {
+        auto beginT = std::chrono::high_resolution_clock::now();
+        alpaka::mem::view::copy(queue, bufHostC, bufAccC, extent);
+        const auto endT = std::chrono::high_resolution_clock::now();
+        std::cout << "Time for HtoD copy: " << std::chrono::duration<double>(endT-beginT).count() << 's' << std::endl;
+    }
 
-    bool resultCorrect(true);
+    int falseResults = 0;
+    static constexpr int NUM_FALSE_RESULTS = 20;
```

Comment:
Perhaps `MAX_NUM_FALSE_RESULTS` would be more descriptive here?

Reply:
It is not a maximum for `falseResults`, but the maximum number to be printed. Maybe `MAX_PRINT_FALSE_RESULTS`?

Reply:
I like your suggestion